seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/JobExecutionIT.java

```diff
@@ -153,6 +153,26 @@ public void testGetErrorInfo() throws ExecutionException, InterruptedException {
         }
     }
 
+    @Test
+    public void testValidJobName() throws ExecutionException, InterruptedException {
+        Common.setDeployMode(DeployMode.CLIENT);
+        String filePath = TestUtils.getResource("valid_job_name.conf");
+        JobConfig jobConfig = new JobConfig();
+        ClientConfig clientConfig = ConfigProvider.locateAndGetClientConfig();
+        clientConfig.setClusterName(TestUtils.getClusterName("JobExecutionIT"));
+        try (SeaTunnelClient engineClient = new SeaTunnelClient(clientConfig)) {
+            ClientJobExecutionEnvironment jobExecutionEnv =
+                    engineClient.createExecutionContext(filePath, jobConfig, SEATUNNEL_CONFIG);
+            final ClientJobProxy clientJobProxy = jobExecutionEnv.execute();
+            CompletableFuture<JobStatus> completableFuture =
+                    CompletableFuture.supplyAsync(clientJobProxy::waitForJobComplete);
+            await().atMost(600000, TimeUnit.MILLISECONDS)
+                    .untilAsserted(() -> Assertions.assertTrue(completableFuture.isDone()));
+            JobResult result = clientJobProxy.getJobResultCache();
+            Assertions.assertEquals("valid_job_name", jobConfig.getName());
```

Comment:
I think we should check name from job status result not job config.
```suggestion
            String value = engineClient.getJobClient().listJobStatus(false);
            Assertions.assertTrue(value.contains("\"jobName\":\"valid_job_name\""));
```